packages/website-backend/src/services/real-time/SseServer.ts

```diff
@@ -0,0 +1,74 @@
+import EventEmitter from 'events';
+import type { Server, ServerResponse } from 'http';
+
+import { AddressInfo } from 'net';
+
+import { SseClient } from './SseClient.js';
+
+export class SseServer extends EventEmitter {
+  public port?: number;
+
+  #server: Server;
+  #cors: string;
+  #clients = new Set<SseClient>();
+
+  constructor(server: Server, cors = '') {
```

Comment:
Could we make `SseServer` injectable and use DI to inject the existing server here? See previous remark.